tests/Aspire.Hosting.Azure.Tests/AzureServiceBusExtensionsTests.cs

```diff
@@ -179,7 +179,7 @@ public async Task VerifyWaitForOnServiceBusEmulatorBlocksDependentResources()
     {
         var cts = new CancellationTokenSource(TimeSpan.FromMinutes(10));
         using var builder = TestDistributedApplicationBuilder.Create(output);
-        
```

Comment:
the whitespace was auto-linted. can revert, but it would be better not to have the extra spaces anyway